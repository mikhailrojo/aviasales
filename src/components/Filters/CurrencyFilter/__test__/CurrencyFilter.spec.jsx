import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {CurrencyFilter} from '../index';

describe('Snapshot', () => {
	it('render CurrencyFilter with selected RUB', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(
			<CurrencyFilter
				changeCurrency={jest.fn()}
				selectedCurrency="RUB"
			/>
		);
		expect(component).toMatchSnapshot();
	});
});
