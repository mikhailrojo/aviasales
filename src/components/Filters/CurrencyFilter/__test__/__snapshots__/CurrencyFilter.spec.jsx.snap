// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render CurrencyFilter with selected RUB 1`] = `
<styled.div>
  <styled.div>
    Валюта
  </styled.div>
  <styled.div>
    <input
      checked={true}
      id="rubCurrency"
      name="currency"
      onChange={[Function]}
      style={
        Object {
          "display": "none",
        }
      }
      type="radio"
      value="RUB"
    />
    <styled.div>
      <label
        htmlFor="rubCurrency"
      >
        RUB
      </label>
    </styled.div>
    <input
      checked={false}
      id="usdCurrency"
      name="currency"
      onChange={[Function]}
      style={
        Object {
          "display": "none",
        }
      }
      type="radio"
      value="USD"
    />
    <styled.div>
      <label
        htmlFor="usdCurrency"
      >
        USD
      </label>
    </styled.div>
    <input
      checked={false}
      id="eurCurrency"
      name="currency"
      onChange={[Function]}
      style={
        Object {
          "display": "none",
        }
      }
      type="radio"
      value="EUR"
    />
    <styled.div>
      <label
        htmlFor="eurCurrency"
      >
        EUR
      </label>
    </styled.div>
  </styled.div>
</styled.div>
`;
